app/design/frontend/default/models/template/user/user.phtml

```diff
@@ -0,0 +1,66 @@
+
+<table class="data-table table-hover" id="table_data" border="">
+  <colgroup><col width="1">
+      <col width="1">
+      <col>
+      <col width="1">
+      <col width="1">
+      <col width="1">
+ </colgroup>
+
+  <tr>
+    <th>USER_Id</th>
+    <th>FIRST NAME</th>
+    <th>LAST NAME</th>
+    <th>EMAIL</th>
+    <th>DOB</th>
+    <th>CITY</th>
+    <th>PINCODE</th>
+    <th>CONTACT</th>
+    <th>Delete</th>
+  </tr>
+ <?php
+    $user_model = $this->getUserCollection();
+ 
+      foreach($user_model as $data)
+        {
+            $user_table_id = $data->getUserId();
+      ?>
+
+           <tr class="first last odd" >
+            <td class="a-center last"  >  <?php echo $data->getUserId() ?>  </td>
+            <td class="a-center last"  >  <?php echo $data->getFirstname() ?> </td>
+            <td class="a-center last"  >  <?php echo $data->getLastname() ?> </td>
+            <td class="a-center last"  >  <?php echo $data->getEmail() ?> </td>
+            <td class="a-center last" >  <?php echo $data->getDob() ?>   </td>
+    <?php
+
+      $address_model = $this->getAddressCollection(); 
+          foreach($address_model as $data)
+            {
+            $address_table_id = $data->getUserId();
+            if($user_table_id == $address_table_id)
+        {
+            ?>
+            <td  class="a-center last" > <?php echo $data->getCity() ?> </td>
+            <td  class="a-center last" > <?php echo $data->getPincode() ?> </td>
+            <td  class="a-center last" > <?php echo $data->getContact() ?> </td>
+          
+          <td><a href='<?php echo $this->getUrl("user/index/delete"); ?>?user_id=<?php echo $data->getUserid() ?>'>Delete</a></td>
+     <?php
+
+              }
+            }
+          }
+    
+    ?>
+
+    <p>
+      <b>WANT TO ADD SOME MORE DATA </b>
```

Comment:
in magento every where always text is written like this
<?php echo __('WANT TO ADD SOME MORE DATA') ?>
follow same format everywhere. see core files to understand